README.md

```diff
@@ -75,7 +75,7 @@ For this tutorial, only the contents of the `/deegree-workshop-bundle` folder in
 | `/deegree-workspace/ps-sl`          | Complete deegree workspace with data and configuration files (WFS, WMS, layers, styles and database) for the INSPIRE Annex 1 data theme ProtectedSites from the federal state of Saarland | [What is a deegree workspace?](https://download.deegree.org/documentation/current/html/#_the_deegree_workspace) |
 | `/sql`      | SQL scripts for setting up the PostgreSQL database                                                                                                                                        | [PostgreSQL](https://www.postgresql.org/docs/current/tutorial.html), [PostGIS](https://postgis.net/workshops/postgis-intro/) |
 | `docker-compose.yaml`         | Docker Compose file for defining and running multi-container applications, in this case including deegree, PostgreSQL and pgAdmin 4                                                       | [How does Docker Compose work?](https://docs.docker.com/compose/compose-application-model/) | 
-| `.env` | Used to set the necessery environment variables                                                                                                                                           | [How to use the `.env`?](https://docs.docker.com/compose/environment-variables/set-environment-variables/) |
+| `.env` | Used to set the necessery environment variables                                                                                                                                           | [How to use the `.env`?](https://docsriables/) |
```

Comment:
Link needs to be restored from `https://docsriables/` to `https://docs.docker.com/compose/environment-variables/set-environment-variables/`.